pkgs/top-level/all-packages.nix

```diff
@@ -1546,6 +1546,10 @@ with pkgs;
 
   fxlinuxprintutil = callPackage ../tools/misc/fxlinuxprintutil { };
 
+  gay = callPackage ../tools/misc/gay {
+    inherit (python38Packages) buildPythonPackage fetchPypi;
```

Comment:
Will `python3Packages` be okay?

Reply:
Should be!

Reply:
i think that line is not needed. see how other packages do it (using pkgs)